### DISCUSSION TITLE
CUE Community Call 2022-11-29

### DISCUSSION THREAD

**Discussion by myitcv** [cue-lang/cue#2152] [Community Calls]
We are excited to confirm Tuesday 29th November at 1630 UTC for our next CUE Community Call!

Please reply with any questions/topics you would like to see covered.

### Agenda

* Project updates
* Release updates
* Follow-ups from [Community Call 2022-10-25](https://github.com/cue-lang/cue/discussions/2033)
* Community Updates
* Spotlight on... [Dagger.io](https://dagger.io/)!

### Meeting details

https://meet.google.com/hib-yqyv-rfr

This session will also be recorded for those unable to join us live.

### CUE Community shared calendar

https://calendar.google.com/calendar/u/0?cid=Y19lNzkxMWQ5OWQ4ZGIyMmU2ZTVjMzhkMTVkNjY2ZTVlNjdiNWE5ODNkZWU4N2JmNTU2NDY3NzI1OGIxYjJhMTFhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20

### Recording

https://youtu.be/Vqe7X_LXIuA

### Slides

https://docs.google.com/presentation/d/1vZnHChGex1zRxtR5cQIQMP19gHzA2xaIjkuAMDnSKr8/edit

**Comment by hugosantos** [comment]
@namespacelabs would be happy to be listed as a Cue user and also happy to go through how we use Cue in a future call! Looking forward to the latest updates.

**Reply by myitcv** [reply]
Hi @hugosantos - I've finally replied to your email! Apologies for the delay

**Comment by phoban01** [comment]
It would be great to hear an update on #2032 if there's any progress to share.

**Reply by myitcv** [reply]
Thanks, @phoban01. @rogpeppe will be providing an update today.

**Comment by myitcv** [comment]
Updated the discussion description with links to the recording and slides.
